### PR TITLE
fix(tree): add translatable strings for selected/unselected node

### PR DESCRIPTION
**Pull request by mivaylo**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-2311

## What is the new behavior?
Easier translation of "selected" and "unselected" strings in tree node

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/angular/src/data/tree-view/tree-node.html [hunk 1]
```diff
@@ -65,8 +65,8 @@
   <ng-template #treenodeContent>
     <ng-content></ng-content>
     <div class="clr-sr-only" *ngIf="featuresService.selectable || ariaSelected">
-      <span *ngIf="ariaSelected"> selected</span>
-      <span *ngIf="!ariaSelected"> unselected</span>
+      <span *ngIf="ariaSelected"> {{selectedString}} </span>
+      <span *ngIf="!ariaSelected"> {{unselectedString}} </span>
```
Comment (valentin-mladenov) [inline projects/angular/src/data/tree-view/tree-node.html:69]: Isn't is better this way?
No need to add geters as well.
```suggestion
      <span>{{ariaSelected ? commonStrings.keys.selectedTreeNode : commonStrings.keys.unselectedTreeNode}}</span>
```

File: projects/angular/src/utils/i18n/common-strings.default.ts [hunk 2]
```diff
@@ -126,4 +126,8 @@ export const commonStringsDefault: ClrCommonStrings = {
   fileCount: '{COUNT} files',
   clearFile: 'Clear {FILE}',
   clearFiles: 'Clear {COUNT} files',
+
+  // Tree node
```
Comment (valentin-mladenov) [inline projects/angular/src/utils/i18n/common-strings.default.ts:130]: `Tree` is enough.
```suggestion
  // Tree
```

File: projects/angular/src/utils/i18n/common-strings.interface.ts [hunk 3]
```diff
@@ -301,4 +301,8 @@ export interface ClrCommonStrings {
   fileCount: string;
   clearFile: string;
   clearFiles: string;
+
+  // Tree node
```
Comment (valentin-mladenov) [inline projects/angular/src/utils/i18n/common-strings.interface.ts:305]: Same as the other.

File: projects/angular/src/data/tree-view/tree-node.html (hunk 1 above)
Comment (mivaylo) [reply projects/angular/src/data/tree-view/tree-node.html:69]: yes, in my opinion looks better that way

**Review by valentin-mladenov — APPROVED** [review]
LGTM

File: projects/angular/src/utils/i18n/common-strings.interface.ts (hunk 3 above)
Comment (mivaylo) [reply projects/angular/src/utils/i18n/common-strings.interface.ts]: Updated in https://github.com/vmware-clarity/ng-clarity/pull/1575/commits/a89de534c1ce7d9bfcc03e67522c1ee5a64de206

File: projects/angular/src/data/tree-view/tree-node.html (hunk 1 above)
Comment (mivaylo) [reply projects/angular/src/data/tree-view/tree-node.html]: Updated in https://github.com/vmware-clarity/ng-clarity/pull/1575/commits/a35bf2586d73d31cf9cb274b679f3f61fc90d45f

**Pull request merged**